com.kaii.hgui.json

```diff
@@ -0,0 +1,50 @@
+{
+    "app-id" : "com.kaii.hgui",
```

Comment:
`kaii.com` doesn't seem to exist.

Reply:
its https://kaii-lb.github.io, wasn't sure how to add it

Reply:
this question remain.

the domain used for the app-id MUST be under the control of the upstream developers.

Reply:
i *am* the upstream developer, the app domain is com.kaii.hgui, the website name is what i said. u asked for kaii.com so i assumed u were talking about a website since it isnt an rDNS

Reply:
![image](https://github.com/flathub/flathub/assets/114441/b8516590-f0bc-42da-807a-a480832a3770)

Reply:
https://kaii-lb.github.io/ its working for me and i tested with a friend on another network

Reply:
then why isn't that used for the app-id?

https://docs.flatpak.org/en/latest/conventions.html#application-ids

Reply:
ah, its an old project and so changing the id would be difficult. and as i undestand `io.github.kaii-lb.hgui` would be invalid?
if anything, remove the website entirely just ignore that it exists

Reply:
the documentation explains what to do with the `-`. *hint*: it involve replacing it with a `_`.

I don't see how changing the id is difficult since it's not yet on flathub.

Reply:
This is a blocker. NO EXCEPTION.

Reply:
alright, did a whole bunch of changes upstream, app id is now `io.github,kaii_lb.hgui` would that be correct? (and the app is building locally thru flatpak-builder)